js/react/components/shared/CustomToast.jsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function CustomToast({ title, type }) {
+	return (
+		<div
+			className={
+				'toast align-items-center text-bg-primary border-0 fade show position-absolute start-50 translate-middle-x ' +
+				(type === 'error' ? 'bg-danger' : 'bg-primary')
+			}
+			style={{ zIndex: 10000, top: '10px', padding: '0.5rem' }}
+			role="alert"
+			aria-live="assertive"
+			aria-atomic="true"
+		>
+			<div className="toast-body text-center">{title}</div>
```

Comment:
This looks good, but I thought we are moving towards using Bootstrap toast instead of custom toast. 
#544 
Correct me if I'm wrong, @Enubia

Reply:
> This looks good, but I thought we are moving towards using Bootstrap toast instead of custom toast. #544 Correct me if I'm wrong, @Enubia

Oh sorry for misleading description of that, I meant Custom Toast as in separated component that uses bootstrap toast classes, specifically these ones https://getbootstrap.com/docs/5.1/components/toasts/#color-schemes. I just removed the dismiss button.

Reply:
@NMilivojevic Oh my bad then. Yeah this is fine. Thanks for creating this.